src/services/videoconference/index.js

```diff
@@ -43,319 +43,314 @@ const { ObjectId } = require('../../helper/compare');
 
 const CLIENT_HOST = Configuration.get('HOST');
 
-class VideoconferenceBaseService {
-	constructor(app) {
-		this.app = app;
-		this.docs = {};
+function throwOnValidationErrors(scopeId, scopeName, options = null) {
+	if (!idAndScopeNameAreValid({ scopeId, scopeName })) {
+		throw new BadRequest('id or scopeName invalid');
 	}
-
-
-	static throwOnValidationErrors(scopeId, scopeName, options = null) {
-		if (!VideoconferenceBaseService.idAndScopeNameAreValid({ scopeId, scopeName })) {
-			throw new BadRequest('id or scopeName invalid');
-		}
-		if (options !== null) {
-			const validOptions = VideoconferenceBaseService.getValidOptions(options);
-			if (!lodash.isEqual(options, validOptions)) {
-				throw new BadRequest('options invalid'); // todo cleanup
-			}
+	if (options !== null) {
+		const validOptions = getValidOptions(options);
+		if (!lodash.isEqual(options, validOptions)) {
+			throw new BadRequest('options invalid'); // todo cleanup
 		}
 	}
+}
 
-	static async throwOnFeaturesDisabled(authenticatedUser) {
-		// throw, if feature has not been enabled
-		if (!Configuration.get('FEATURE_VIDEOCONFERENCE_ENABLED') === true) {
-			throw new Forbidden('feature FEATURE_VIDEOCONFERENCE_ENABLED disabled');
-		}
-		// throw, if current users school feature is not enabled
-		const schoolFeatureEnabled = await VideoconferenceBaseService
-			.isSchoolFeatureEnabled(authenticatedUser.schoolId);
-		if (!schoolFeatureEnabled) {
-			throw new Forbidden('school feature disabled');
-		}
+async function throwOnFeaturesDisabled(authenticatedUser) {
+	// throw, if feature has not been enabled
+	if (!Configuration.get('FEATURE_VIDEOCONFERENCE_ENABLED') === true) {
+		throw new Forbidden('feature FEATURE_VIDEOCONFERENCE_ENABLED disabled');
 	}
+	// throw, if current users school feature is not enabled
+	const schoolFeatureEnabled = await isSchoolFeatureEnabled(authenticatedUser.schoolId);
+	if (!schoolFeatureEnabled) {
+		throw new Forbidden('school feature disabled');
+	}
+}
 
-	static throwOnPermissionMissingInScope(permission, permissions) {
-		if (!VideoconferenceBaseService.userIsAllowedTo(permission, permissions)) {
-			throw new Forbidden(`permission ${permission} not given in scope`);
-		}
+function throwOnPermissionMissingInScope(permission, permissions) {
+	if (!userIsAllowedTo(permission, permissions)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['userIsAllowedTo' was used before it was defined.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5401970)